apps/web/sdk/index.ts

```diff
@@ -12,14 +13,25 @@ export const useSdk = createSharedComposable(() => {
     }),
   };
 
-  client.interceptors.request.use(
+  const headers = useRequestHeaders(['cookie']);
+
+  // If an interceptor is already set, eject it to remove it
+  // this prevents headers being shared across instances
+  if (interceptorId !== null) {
+    client.interceptors.request.eject(interceptorId);
+  }
```

Comment:
**suggestion:** Extract into its own method and give the method an expressive name, so that we no longer need the comment.